website/src/pages/home/_Features.tsx

```diff
@@ -0,0 +1,123 @@
+            </>
+        ),
+        details: [
+            'Update legacy code, remove outdated patterns, and fix critical security issues across multiple repositories',
+            'Track changes across your codebase without maintaining spreadsheets of places that need to be updated',
+            'Automate fixing breaking changes introduced by library or package updates',
+        ],
+        ctaLink: '/batch-changes',
+    },
+    {
+        productFeature: 'code insights',
+        title: 'Track what really matters to you and your team',
+        description:
+            'With Code Insights, know how different initiatives in your codebase are progressing in real time so you can make data-driven decisions.',
+        details: [
+            'Track migrations, code smells, ownership, versions, and fully customizable metrics',
+            'Ensure that security vulnerabilities and deprecated packages get completely removed from your codebase',
+            'Create visualizations that update automatically, provide historical trends, and pull directly from the source of truth: the code itself',
+        ],
+        ctaLink: '/code-insights',
+    },
+]
+
+const FeatureSection: FunctionComponent = () => (
+    <ContentSection color="white" className="py-8 mt-8 pb-7">
+        <div className="text-center">
+            <h1 className="font-weight-bold">How teams use Sourcegraph</h1>
+            <p className="max-w-500 mx-auto">
+                Sourcegraph's code intelligence platform is built with features that help you understand, fix, and
+                automate across your entire codebase.
+            </p>
+        </div>
+
+        {features.map((feature, i) => (
+            <div
+                key={i}
+                className={classNames('row py-7', {
+                    'flex-lg-row-reverse': i % 2,
+                })}
+            >
+                <div
+                    className={classNames('col-lg-6', {
+                        'pr-lg-7 pl-lg-0': i % 2 === 0,
+                        'pl-lg-7 pr-lg-0': i % 2,
+                    })}
+                >
+                    <small className="text-uppercase font-weight-bold d-block mb-2">{feature.productFeature}</small>
+                    <h2 className="font-weight-bold">{feature.title}</h2>
+                    <p>{feature.description}</p>
+                    <ul>
+                        {feature.details.map((detail, k) => (
+                            <li key={k}>{detail}</li>
+                        ))}
+                    </ul>
+                    {feature.ctaLink.includes('http') ? (
+                        <a href={feature.ctaLink} target="_blank" rel="noreferrer">
+                            Learn more
+                        </a>
+                    ) : (
+                        <Link to={feature.ctaLink}>Learn more</Link>
+                    )}
+                </div>
+
+                <div className="col-lg-6 mt-3 mt-lg-0 p-0">
```

Comment:
From the spec, the image/animation should be above the text on mobile. So maybe just put this block above the text section & adjust the `flex-lg-row-reverse` logic?

I know the image is just a placeholder & this will be adjusted more. Just flagging to give the image/animation padding on mobile. The figma may be incomplete here, I have no intel on what the animation will be exactly.

Reply:
Yeah that's a good idea! Thanks for catching this. Updated in [this commit](https://github.com/sourcegraph/about/pull/5276/commits/f9602696dfa867277575f8152b3c14dacc1ff725)

Reply:
Since I'm using flex reverse row/column, I'm reversing in CSS instead of moving the markup so I have more control at different breakpoints.